src/App.vue

```diff
@@ -388,6 +393,17 @@ export default {
     goToNotificationSettings () {
       this.goToSettingsSection()
       this.$router.push({ name: 'Users' })
+    },
+    updateSystemData () {
+      if (localStorage.getItem('token')) {
+        this.updateIsUserLogged(true)

```

Comment:
we can not load all this here, only if I go to the list of agents we need to load the agents, the same with targets and users or other resources, this is really bad for performance